gulpfile.js

```diff
@@ -226,7 +226,7 @@ function getModulesListToAddInBanner(modules) {
 }
 
 function gulpBundle(dev) {
-  return bundle(dev).pipe(gulp.dest('build/' + (dev ? 'dev' : 'dist')));
+  return bundle(dev, undefined, dev ? helpers.getDevPath() : helpers.getBuildPath()).pipe(gulp.dest('build/' + (dev ? 'dev' : 'dist')));
```

Comment:
does this do anything? `bundle` only takes two arguments?

https://github.com/prebid/Prebid.js/blob/d16c038c38fa508ed8cca3489c8f958ba056cb15/gulpfile.js#L308